CHANGELOG

```diff
@@ -1,3 +1,11 @@
+2016-03-04: v0.5.2
+  Add EnforceRFC952 to sample config
+  Improve performance of record generation
```

Comment:
I can try to tackle that next week. Want to assign me a ticket?

On Fri, Mar 4, 2016 at 6:42 PM, Sargun Dhillon <notifications@github.com>
wrote:

> In CHANGELOG
> <https://github.com/mesosphere/mesos-dns/pull/407#discussion_r55105595>:
>
> > @@ -1,3 +1,11 @@
> > +2016-03-04: v0.5.2
> > +  Add EnforceRFC952 to sample config
> > +  Improve performance of record generation
>
> Do you want to do that before we proceed? I primarily did this PR to get
> the build for testing. We can stall if you don't feel comfortable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/407/files#r55105595>.
>